packages/pn-personafisica-webapp/src/components/Notifications/DesktopNotifications.tsx

```diff
@@ -192,7 +194,30 @@ const DesktopNotifications = ({
           testId="notificationsTable"
         />
       ) : (
-        <EmptyState {...EmptyStateProps} />
+        <EmptyState
+          sentimentIcon={filtersApplied ? KnownSentiment.DISSATISFIED : KnownSentiment.NONE}
+        >
+          {filtersApplied && (
+            <Trans
```

Comment:
I'm not sure that all the previous condition in EmptyStateProps are now respected